modules/serverless-creating-a-kafka-event-sink.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * serverless/develop/serverless-event-sinks.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-creating-a-Kafka-event-sink_{context}"]
+
+= Creating a Kafka event sink
+
+As a developer, you can create an event sink to receive events from a particular source and send them to a kafka topic. 
+
+.Prerequisites
+* You have installed the Red Hat OpenShift Serverless operator, with Knative Serving, Knative Eventing, and Knative Kafka APIs, from the Operator Hub.
+
+* You have created a Kafka topic in your Kafka environment.
+
+.Procedure
+. In the developer perspective, navigate to the *+Add* view.
+. Click *Event Sink* in the *Eventing catalog*.
+. Search for KafkaSink in the catalog items and click it.
+. Click *Create Event Sink*.
+. In the form view, type the URL of the bootstrap server, which is a combination of host name and port. 
++
+image::create-event-sink.png[]
+
+. Type the name of the topic to send event data.
+. Type the name of the event sink.
+. Click *Create*. 
+
+.Verification
+. In the developer perspective, navigate to the *Topology* view.
```

Comment:
s/developer perspective/Developer perspective should be in bold. Anything that you see in the GUI should be in Bold. If a user needs to fill up a term, say in a form or something, it should be in back ticks